modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/petra/sql/dsl/expression/OrderByExpressionUtil.java

```diff
@@ -47,8 +49,24 @@ public static OrderByExpression[] getOrderByExpressions(
 					fieldName = parts[1];
 				}
 
-				Column<?, ?> column = objectFieldLocalService.getColumn(
-					objectDefinitionId, fieldName);
+				Column<?, ?> column = null;
+
+				try {
+					column = objectFieldLocalService.getColumn(
+						objectDefinitionId, fieldName);
+				}
+				catch (UnsupportedOperationException
+							unsupportedOperationException) {
+
+					Table<?> table = objectFieldLocalService.getTable(
+						objectDefinitionId, fieldName);
+
+					column = table.getColumn(fieldName + StringPool.UNDERLINE);
+
+					if (column == null) {
+						throw unsupportedOperationException;
```

Comment:
@TaiPhamLR @sontruongces could you explain the reason why you added this code?

Reply:
@sontruongces had created PR wrong branch. sorry about that. He'll create new PR